__tests__/app.test.js

```diff
@@ -191,4 +191,35 @@ describe("PATCH /api/review/:review_id", () => {
         });
    })  
     
+});
+
+//A test battery that ensure that the user endpoint responds with an array the appropirate objects
+describe("GET /api/users", () => {
+    test("status 200: responds with an array of user objects containing username, name and avatar_url", () => {
+        return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then (({ body }) => {
+            const {users} = body;
+            expect(users).toBeInstanceOf(Array);
+            expect(users).toHaveLength(4);
+            users.forEach((user) => {
+                expect(user).toEqual(expect.objectContaining(
+                    {
+                    username: expect.any(String),
+                    name: expect.any(String),
+                    avatar_url: expect.any(String),
+                    }
+                ));
+            });
+        });
+    });
+    test("Status 404: the user inputs a page that doesn't exist", () => {
```

Comment:
I would honestly move this test to a single describe block or just have it somewhere once as its the same test you have further up. Its not really pertinent to this end point specifcally

Reply:
Ok will create a new describe block for this test - do you mean remove it from both places where it currently is and just have one describe by itself, or just have it in the the first block it's currently in?